metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -1149,7 +1150,10 @@ def execute_graphql(
         )
         result = self._post_generic(url, body)
         if result.get("errors"):
-            raise GraphError(f"Error executing graphql query: {result['errors']}")
+            if format_exception:
+                raise GraphError(f"Error executing graphql query: {result['errors']}")
+            else:
+                raise GraphError(result["errors"])
```

Comment:
the "right" way to do this would be to create a `GraphqlError` that has a `.errors` attribute

Reply:
it would have a nice `__str__` method, but would still let us introspect the raw error info